docker-compose.yaml

```diff
@@ -0,0 +1,28 @@
+version: '3.8'
+
+services:
+  backend:
+    build: .
+    container_name: healoscope-backend
+    ports:
+      - "8080:8080"
+    environment:
+      - PORT=8080
+      - MONGO_URI=mongodb://mongo:27017/healoscope
+    depends_on:
+      - mongo
+    volumes:
+      - .:/app
```

Comment:
not needed for backend service. instead create a `.dockerignore` file and mention

```.dockerignore
node_modules/
package-lock.json
```